ccmlib/scylla_node.py

```diff
@@ -242,7 +242,7 @@ def _start_scylla(self, args, marks, update_pid, wait_other_notice,
         if wait_for_binary_proto:
             self.wait_for_binary_interface(from_mark=self.mark, process=self._process_scylla)
         else:
-            time.sleep(2)
+            time.sleep(0.1)
```

Comment:
this is without a loop its changing the behavior - not sure it will not cause tests to fail on slow machines

Reply:
This is a nowait branch. There should be no sleep at all. Whoever added it could have had any motives, but we wouldn't know if it is needed at all without removing it.

Reply:
AFAIK no_wait means you do not wait for cql port to be open ..., so you can't remove the sleep